optimade/server/routers/static/landing_page.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        {% block head %}
+        <title>{{ optimade_title }}</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="keywords" content="optimade,materials,crystals,database">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        {% endblock %}
+
+    <style>
+        span.version {background-color: lightgrey; font-weight: bold; font-family: monospace; padding: 0em 0.5em 0em 0.5em; border-radius: 1em;}
+    </style>
+    </head>
+    <body>
+        <h3>This is an <a href="https://optimade.org">OPTiMaDe</a> base URL running which can be queried with an OPTiMaDe client.</h3>
```

Comment:
The OPTiMaDe webpage doesn't work unless `www` is added.

```suggestion
        <h3>This is an <a href="https://www.optimade.org">OPTiMaDe</a> base URL which can be queried with an OPTiMaDe client.</h3>
```

Reply:
Damn, thought I tried this but I must have tried http rather than https, nice spot.